src/java/org/apache/cassandra/io/sstable/CQLSSTableWriter.java

```diff
@@ -492,30 +496,50 @@ public Builder using(String modificationStatement)
             return this;
         }
 
+        /**
+         * Defines the maximum SSTable size in mebibytes when using the sorted writer.
+         * By default, i.e. not specified, there is no maximum size limit for the produced SSTable
+         *
+         * @param size the maximum sizein mebibytes of each individual SSTable allowed
+         * @return this builder
+         */
+        public Builder withMaxSSTableSizeInMiB(int size)
+        {
+            if (size <= 0)
```

Comment:
I'd prefer "<" instead of "<=" while keeping "0" for representing another state, maybe in future.